Out of Memory FAQ (#2061)
rcgsheffield/sheffield_hpc: FAQs.rst

|br|

"Out of Memory", "OOM" errors and job prematurely stopping
----------------------------------------------------------

> When "Out of Memory" (OOM) errors occur in an interactive or batch  session , it indicates insufficient memory has been allocated for the job to run to completion.

Reviewer: ```suggestion
When "Out of Memory" (OOM) errors occur in an interactive or batch session, it indicates insufficient memory has been allocated for the job to run to completion.
```

> For both interactive and batch jobs, resources are requested for the job. When 'Out of Memory' and 'OOM' errors occur it means not enough memory has been allocated for the job to run to completion.

Reviewer 2: ```suggestion
When "Out of Memory" (OOM) errors occur in an interactive or batch  session , it indicates insufficient memory has been allocated for the job to run to completion.
```

> |br|

Reviewer 3: ```suggestion
-----
```

Looks like we don't really need the additional break line but we are missing a separator.
